optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/src/main/java/org/optaplanner/workbench/screens/domaineditor/client/widgets/planner/PlannerDataObjectEditor.java

```diff
@@ -234,10 +234,12 @@ public void callback(List<Path> paths) {
                     if (lastObjectPropertyInPath.isBaseType() || lastObjectPropertyInPath.isPrimitiveType()) {
                         throw new IllegalStateException(dataObject.getName() + "Comparator: Cannot append property " + objectPropertyName + " to primitive/base type " + lastObjectPropertyInPath.getClassName());
                     }
+
                     DataObject lastDataObjectInPath = getContext().getDataModel().getDataObject(lastObjectPropertyInPath.getClassName());
-                    if (lastObjectPropertyInPath == null) {
-                        throw new IllegalStateException(dataObject.getName() + "Comparator: Data object " + lastObjectPropertyInPath.getClassName() + " not found");
+                    if (lastDataObjectInPath == null) {
+                        throw new IllegalStateException(dataObject.getName() + "Comparator: Data object not found");
```

Comment:
@rsynek `lastObjectPropertyInPath.getClassName()` is the name of the data object that couldn't be found. Please keep it in the exception message. It is a significant piece of information.

Reply:
@yurloc thanks, this is the result of me not being anyhow familiar with this codebase yet. Hopefully nothing more slipped my attention now.